pkg/scheduler/ugm/manager_test.go

```diff
@@ -332,7 +332,7 @@ func TestUpdateConfig(t *testing.T) {
 func TestUseWildCard(t *testing.T) {
 	setupUGM()
 	manager := GetUserManager()
-	user := security.UserGroup{User: "user1", Groups: []string{"group1"}}
+	user1 := security.UserGroup{User: "user1", Groups: []string{"group1"}}
```

Comment:
Did a little cleanup here, variables were inconsistent.
user -> user1
user1 -> user2

It's a problem in multiple tests, which can be taken care of in a separate JIRA.

Reply:
It is a preference: Equals is slightly easier to use as you specify the exact value to compare against and fail the test if it is not the value. Assert fails if the 2nd argument after evaluation is False.

The case for using Equals with boolean is when comparing to False to prevent the negation of the 2nd argument. Usage of `!` in the Assert is easily overlooked, especially if the term becomes more complex.